src/fides/api/alembic/migrations/versions/81886da90395_add_legal_basis_dimension.py

```diff
@@ -4,7 +4,7 @@
 need to instead save preferences against a purpose/vendor AND a legal basis.
 
 Revision ID: 81886da90395
-Revises: 4cb3b5af4160
+Revises: 9b98aba5bba8
```

Comment:
just patching up migration comment in previous migration to keep a clean paper trail 👍

Reply:
Ah thank you 🙏